capability.md

```diff
@@ -0,0 +1,60 @@
+Capability
+==========
+
+Capability Query
+---
+
+The capability query provides a list of all modes supported by each pin and the resolution used by each pin. Each pin has 2 bytes for each supported mode and a value of 127 to mark the end of that pin's data. The number of pins supported is inferred by the message length.
+
+### Capabilities query
+```c++
+0  START_SYSEX              (0xF0)
+1  CAPABILITY_QUERY         (0x6B)
+2  END_SYSEX                (0xF7)
+```
+
+### Capabilities response
+```c++
+0  START_SYSEX              (0xF0)
+1  CAPABILITY_RESPONSE      (0x6C)
+2  1st supported mode of pin 0
+3  1st mode's resolution of pin 0
+4  2nd supported mode of pin 0
+5  2nd mode's resolution of pin 0
+... additional modes/resolutions, followed by PIN_MODE_IGNORE,
+    to mark the end of the pin's modes. Subsequently, each pin
+    follows with its modes/resolutions and a PIN_MODE_IGNORE,
+    until all pins are implemented.
+N  END_SYSEX                (0xF7)
+```
+
+#### Supported Modes
```

Comment:
I'm going to deprecate a few of these modes, leaving only pin modes relate to actual hardware functionality. So Servo, Shift, OneWire, Stepper, and Encoder will be deprecated as pin modes. A new[ feature query](https://github.com/firmata/protocol/pull/47) will be added instead.